src/cba-list-new/cba-list-new.js

```diff
@@ -22,9 +22,10 @@ class List extends HTMLElement
     this.attachShadow({mode: "open"});
     this.shadowRoot.innerHTML = `
     <div id="container">
-      <h2></h2>
-      <h3 id="column"><a href="#"></a></h3>
```

Comment:
As we have removed the `column` element we might want to remove logic and styles associated with it.